torch/fx/experimental/sym_node.py

```diff
@@ -718,10 +718,33 @@ def _sympy_ite(a, t, f):
 
 
 def _get_sym_math_fn(name):
+    cache = None
+
     def fn(a):
-        import sympy
+        nonlocal cache
+
+        if cache is None:
+            import sympy
+
+            class OpaqueUnaryFn(sympy.Function):
+                _torch_handler_name = name
+
+                @classmethod
+                def eval(cls, a):
+                    if isinstance(a, (sympy.Integer, sympy.Float)):
+                        # Python converts to float64 before computing, c.f.
+                        # >>> math.sin(2**53+1)
+                        # -0.848925964814655
+                        # >>> math.sin(float(2**53+1))
+                        # -0.848925964814655
+                        return sympy.Float(getattr(math, name)(float(a)))
+                    return None
```

Comment:
curiosity q: should this be an error? Or can we ever pass in something that is not a Sympy int/float + want to return None here

Reply:
The meaning of None in eval was "did not simplify, no change". So it's appropriate, no need to argument validate.